### PR TITLE
[MLIR][LLVM] Cleanup attr-dict printing (NFC)

### PR DESCRIPTION
**Pull request by gysit**
This commit simplifies the custom attribute dictionary printing and uses it only for printing ops that have fast math flags.

### CODE REVIEW

File: mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td [hunk 1]
```diff
@@ -524,7 +523,6 @@ class LLVM_CastOpWithNNegFlag<string mnemonic, string instName, Type type,
     $res = op;
   }];
 }
-
```
Comment (Dinistro) [inline mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td:527]: Nit: Seems unrelated

**Review by Dinistro — APPROVED** [review]
Should this be marked as NFC?
Otherwise, LGTM!

File: mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td (hunk 1 above)
Comment (gysit) [reply mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td:527]: Yes it is a small format cleanup...
